hazelcast-client/src/test/java/com/hazelcast/client/map/ClientMapBasicTest.java

```diff
@@ -54,13 +54,23 @@
 @RunWith(HazelcastParallelClassRunner.class)
 @Category(QuickTest.class)
 public class ClientMapBasicTest {
+    private static final String KEY = "key";
+    private static final String OLD_VALUE = "oldValue";
+    private static final String NEW_VALUE = "newValue";
 
-    static HazelcastInstance client;
-    static HazelcastInstance server;
+    private static HazelcastInstance client;
+    private static HazelcastInstance server1;
+    private static HazelcastInstance server2;
+    private static HazelcastInstance server3;
+
+    private String randomMapName = randomString();
+    private IMap<String, String> randomStringMap = client.getMap(randomMapName);
 
```

Comment:
I just did a check check with another test, it appears that a new instance is created for every test method. So there is no shared state between fields. So I was wrong about that this approach is racy. Well.. I learned something new today.

```
 @Test
    @ClientCompatibleTest
    public void testSet(){
        HazelcastInstance hzInstance = createHazelcastInstance();
        IAtomicLong atomicNumber = hzInstance.getAtomicLong(randomString());
        atomicNumber.set(271);
        assertEquals(271,atomicNumber.get());

        System.out.println("testSet:"+System.identityHashCode(this));
    }

    @Test
    @ClientCompatibleTest
    public void testGet(){
        HazelcastInstance hzInstance = createHazelcastInstance();
        IAtomicLong atomicNumber = hzInstance.getAtomicLong(randomString());
        assertEquals(0,atomicNumber.get());
        System.out.println("testGet:"+System.identityHashCode(this));

    }

arted Running Test: AtomicLongTest.testSet
Started Running Test: AtomicLongTest.testGet
ASSERT_TRUE_EVENTUALLY_TIMEOUT = 120
testGet:1332226643
testSet:1068685279
Started Running Test: AtomicLongTest.testSet
Started Running Test: AtomicLongTest.testGet
ASSERT_TRUE_EVENTUALLY_TIMEOUT = 120
testGet:1332226643
testSet:1068685279
Finished Running Test: AtomicLongTest.testSet in 0.836 seconds.
```